### ISSUE TITLE
Asura Scans: HTTP error 404

### ISSUE THREAD

**Issue by SrW0nka** [tachiyomiorg/tachiyomi-extensions#11104]
### Source information

Asura scans 1.2.24

### Steps to reproduce

1. Open extension main page or any added title

### Expected behavior

1. Load 

### Actual behavior

Error 404

### Tachiyomi version

0.13.1

### Android version

A11

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by sovit042** [comment]
The same problem is happening to me 😭😭

**Comment by connect192168** [comment]
I'm having the same issue (tachiyomi 0.13.1 as above and the latest asurascans extension version as of posting),
Tried the troubleshooting guide and a different public IP and no change in when tachiyomi get a 404 error compared to before I tried fixing things (same steps as above, but also for downloading chapters).

Opening the site in a browser works fine, so I'd guess it's probably a site structure change.

Wish I had the skills and insight to fix it myself, thanks in advance to anyone who can help and contribute to fixing the issue.
